### PR TITLE
added autostart for vm into virt

### PR DESCRIPTION
**Pull request by smolaon**

### CODE REVIEW

File: salt/control/virt.sls [hunk 1]
```diff
@@ -67,6 +67,14 @@ salt_control_virt_{{ cluster_name }}_{{ node_name }}:
 #  - path: /srv/salt-images/{{ node_name }}.{{ cluster.domain }}/system.qcow2
 #  - id_: {{ node_name }}.{{ cluster.domain }}
 #  - unless: virsh list | grep {{ node_name }}.{{ cluster.domain }}
+
+salt_virt_autostart_{{ cluster_name }}_{{ node_name }}:
```
Comment (cznewt) [inline salt/control/virt.sls]: Where is the README covering this? :)

Comment (smolaon) [reply salt/control/virt.sls]: Yes, thats good idea. Dont merge this.

Comment (fpytloun) [inline salt/control/virt.sls]: It should be also controlled by some parameter, eg. `node.get('autostart', True)`

Comment (cznewt) [reply salt/control/virt.sls]: yes, he was told already + status which enforces it right away

**Pull request merged**